apps/frontend/src/app/3_organisms/GettingStartedPopup/GettingStartedPopup.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { FC, useCallback, useEffect, useMemo, useState } from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { reactLocalStorage } from 'reactjs-localstorage';
+
+import {
+  Button,
+  ButtonType,
+  Checkbox,
+  Dialog,
+  Heading,
+  HeadingType,
+  Link,
+  Pagination,
+} from '@sovryn/ui';
+
+import { useIsMobile } from '../../../hooks/useIsMobile';
+import { translations } from '../../../locales/i18n';
+import { sovrynWikiLinks } from '../../../utils/constants';
+import styles from './GettingStartedPopup.module.css';
+
+const DEFAULT_PAGE_SIZE = 1;
+
+export const GettingStartedPopup: FC = () => {
+  const { t } = useTranslation();
+  const { isMobile } = useIsMobile();
+  const [page, setPage] = useState(0);
+  const [isOpen, setIsOpen] = useState(false);
+  const [checked, setChecked] = useState(false);
+  const pageOffset = useMemo(() => page + DEFAULT_PAGE_SIZE, [page]);
+
+  const data = useMemo(
+    () => [
+      {
+        name: t(translations.gettingStartedPopup.content.pool.title),
+        description: t(
+          translations.gettingStartedPopup.content.pool.description,
+        ),
+        href: sovrynWikiLinks.pool,
+      },
+      {
+        name: t(translations.gettingStartedPopup.content.lend.title),
+        description: t(
+          translations.gettingStartedPopup.content.lend.description,
+        ),
+        href: sovrynWikiLinks.lend,
+      },
+      {
+        name: t(translations.gettingStartedPopup.content.trade.title),
+        description: t(
+          translations.gettingStartedPopup.content.trade.description,
+        ),
+        href: sovrynWikiLinks.trade,
+      },
+      {
+        name: t(translations.gettingStartedPopup.content.bridge.title),
+        description: t(
+          translations.gettingStartedPopup.content.bridge.description,
+        ),
+        href: sovrynWikiLinks.bridge,
+      },
+    ],
+    [t],
+  );
+
+  const items = useMemo(
+    () => (isMobile ? data.slice(page, pageOffset) : data),
+    [data, pageOffset, isMobile, page],
+  );
+
+  const onClick = useCallback(() => {
+    if (checked) {
+      reactLocalStorage.set('gettingStartedPopup', 'false');
```

Comment:
Extract `gettingStartedPopup` to a variable (e.g. `localStorageKey`) and use that instead.